src/utils/utils.ts

```diff
@@ -75,57 +87,76 @@ export async function createTemplateApp(folderPath: string, template: string) {
       return reject(new Error(`Unable to fetch template "${template}"`))
     }
 
-    createApp(folderPath, `https://github.com/sasjs/template_${template}.git`)
+    createApp(
+      folderPath,
+      `https://github.com/sasjs/template_${template}.git`,
+      'https://github.com/sasjs/docs'
+    )
     return resolve()
   })
 }
 
+/**
+ * It will download and unzip `sasjs/docs` into newly created folder into `docs` subfolder
```

Comment:
This function will first download / unzip the target app from `repoUrl`, then download / unzip the `sasjs/docs` repo into a subdirectory named 'docs'.  

Finally, if `installDependencies` is `true`, the `npm install` command will be executed.